Stripe/STPPaymentContext.m

```diff
@@ -145,6 +146,7 @@ - (void)retryLoading {
                 [self.loadingPromise fail:error];
                 return;
             }
+            self.shippingAddress = customer.shipping;
```

Comment:
Ah, yep – making some additional updates here also to allow prefilling `shippingAddress` programmatically. I think the behavior should probably be: don't skip the "show shipping VC" step of request payment if:
  - the address is incomplete
  - the user hasn't verified the address
  - the delegate implements `paymentContext:didUpdateShippingAddress:` and `selectedShippingMethod` is nil